### PR TITLE
Fix calculated width of prompt with ansi formatting

### PR DESCRIPTION
**Pull request by gwenn**

### CODE REVIEW

**Comment by gwenn** [comment]
Fix #22

File: examples/example.rs [hunk 1]
```diff
@@ -12,7 +12,7 @@ fn main() {
         println!("No previous history.");
     }
     loop {
-        let readline = rl.readline(">> ");
+        let readline = rl.readline("\x1b[1;32m>>\x1b[0m ");
```
Comment (kkawakam) [inline examples/example.rs]: Is it really necessary to have this in the example?

Comment (gwenn) [inline examples/example.rs]: No, it is not necessary.
It just shows that rustyline properly handles prompt with ansi escape codes.

**Comment by kkawakam** [comment]
Thanks for the fix.

**Pull request merged**